scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -330,13 +320,14 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
                                            disable_override_desc="Skip the building of the O3DE toolchain executables (AssetBundlerBatch, AssetProcessorBatch) as part of the export process. "
                                                                  "The tools must be already available at the location specified by the '--tools-build-path' argument.")
 
-        default_build_tools_path = export_config.get_value(exp.SETTINGS_DEFAULT_BUILD_TOOLS_PATH.key, exp.SETTINGS_DEFAULT_BUILD_TOOLS_PATH.default)
-        parser.add_argument('-tbp', '--tools-build-path', type=pathlib.Path, default=pathlib.Path(default_build_tools_path),
-                            help=f'Designates where the build files for the O3DE toolchain are generated. If not specified, default is {default_build_tools_path}')
+        
+        # Do not supply a default path for the tools build path. This should be inferred by the export script based on engine installation.
```

Comment:
I understand this is to handle the SDK Installer case, but isn't this still relevant for source builds?

Reply:
That's part of the reason to leave it blank at the arguments level. The actual export function will infer the tools build path based on whether the engine is installer or source. We're moving the burden of defaults to the export function in essence, b/c we have to check what kind of engine we're dealing with anyways.

Reply:
What is the inferred path for source builds then vs SDK builds

Reply:
inferred path for source builds is _<project_folder>/build/tools_, no?

Reply:
upon further review, I think my test cases sufficiently cover for the scenario where the basic default value is provided. As such, I'm reverting these particular changes for the default build paths.